server/aurora_mcp/chat_bridge.py

```
@@ -0,0 +1,188 @@
+
+_TERMINAL_OK = frozenset({"complete", "completed"})
+_TERMINAL_ERR = frozenset({"error", "cancelled", "failed"})
+
+# Canonical assistant-message sender tag in chat_sessions.messages is "bot"
+# (see DB). Accept "aurora" too in case the schema ever shifts.
+_ASSISTANT_SENDERS = frozenset({"bot", "aurora"})
+
+ApiCall = Callable[..., Awaitable[Dict[str, Any]]]
+
+
+def _validate_inputs(message: Any, session_id: Optional[str], mode: str, poll_only: bool) -> Optional[Dict[str, Any]]:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `message`

(ANN401)